public/css/reader-app.css

```diff
@@ -152,6 +152,14 @@
 
 ul.thread-list.reader .message {
     background-color: black;
+    padding: 8px 10px;
+    margin: 2px 0;
+}
+ul.thread-list.reader strong {
+    color: #fff;
+}
+ul.thread-list.reader li {
+    padding: 10px 0;
```

Comment:
I don't understand this line. Isn't this rule already applied, from [common.css:241](https://github.com/emanchado/narrows/blob/master/public/css/common.css#L238-L242)?

Reply:
Oh, haha, of course! Sorry, I don't know what I was thinking :sweat_smile: